community/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,123 @@
+with the SourceCred community. This code of conduct is especially relevant for any
+group or individual that is representing the SourceCred community in an event
+(i.e., as speaker, volunteer, or other staff), and should govern governs standards of behaviors
+for the individual's participation in said event.
+
+This code of conduct is not exhaustive or complete, but rather serves to distill our understanding of a collaborative, inclusive community, and to provide avenues for seeking support if this
+code is breached. We should try to follow this code in spirit as much as in letter, to create a friendly and productive environment that enriches the SourceCred community.
+
+Unless explicitly specified for a particular project or initiative, this code
+of conduct serves as a global standard for all SourceCred affiliated projects, activities,
+and events. If an individual is found to violate this Code of Conduct, the Benevolent
+Dictator may recruit an advisory committee from the community and tkae any action they
+deem appropriate to resolve the situation including warning the person in question,
+removing them from a specific activity, or banning them from further participation in 
+activities (in person or online). The last option is unlikely, but possible in extreme
+circumstances.
+
+<br>
+
+## Standards for Behavior
+
+All communication should be appropriate for a professional audience including people of many different backgrounds. Please be yourself, but be respectful of others, and always be kind and courteous. There is never a need to be mean, rude, or disrespectful. Specifically, the SourceCred community strives to:
+
+ - **Be empathetic**. Realize that every post, event, project, or program is created by human beings that are likely trying their best, and aren't perfect. Try your best to understand perspectives that differ from your own, and to ask for clarification when you don't understand how someone feels.
+ - **Be respectful**: Every person deserves to be treated with kindness and respect. If we assume good intentions, we can work together to resolve conflict. We may all experience some frustration from time to time, but we do not allow frustration to turn into a personal attack. A community where
+people feel uncomfortable or threatened is not a productive one.
+ - **Be welcoming**: When an interaction or community is new, it can be intimidating. We aspire to be welcoming, friendly, and patient to both new and current members, and remember that everyone is a constantly growing and learning at all points in our lives.
+ - **Be vulnerable**. If you feel the compulsion to act in a disrespectful or otherwise inappropriate manner, you should practice self-compassion, and both step away from the current interaction, and seek others for support. It can be helpful to talk to someone about what is troubling you, and discuss how you can approach a situation with respect, and without conflict. 
```

Comment:
Mixed feelings on including vulnerability here. While it is certainly a valuable practice that can lead to more skillful interactions, telling people to be vulnerable might be problematic for a couple reasons: a) sometimes it's not safe to be vulnerable, especially if you're new in a community and still feeling out its norms and getting to know people, and b) it's sadly a somewhat "advanced" skill in many cultures (e.g. the US). I've personally been working on this for years, and it's still something I can't always bring myself to do (or do well). If we create the expectation that people be "vulnerable", what does that mean? At what level? In what ways? Am I vulnerable enough? There could be some big cultural differences in interpretation too. 

@Lex-Birdie and I were recently talking about how to create consent culture, perhaps education materials to help people learn more advanced communication skills, even creating ways people can prove to the community they have demonstrated certain knowledge and skills ([notes of convo](Lex-Birdie)). I wonder if topics like vulnerability might be better explored in other docs?

Reply:
I think I wrote the bullet and then tried to come up with a good descriptor for it, and it seemed appropriate at the time. What about "be compassionate?" or "be self-aware?" (or something around awareness).

Reply:
Upon re-reading, it seems like the bullet is actually more about being self aware and compassionate than vulnerability, per se. We already have "Be aware" as the descriptor of another bullet point. "Be compassionate" works. I think another common descriptor that fits this is "Respond not react", or "Don't react, respond"? "Respond skillfully"? "Be aware of your reactions"?

Reply:
I'm good with Be compassionate, mostly because it matches the format of the others, and hints at self compassion and compassion for the other person. What do others think?

Reply:
Works for me.